fontforge/splinestroke.c

```diff
@@ -864,7 +864,7 @@ static void HideStrokePointsCircle(StrokeContext *c) {
 		if ( dist1sq<400 )
 		    j -= (6-1);
 		else
-		    j -= .66667*sqrt(dist1sq)-1;
+		    j -= .5*sqrt(dist1sq)-1;
```

Comment:
It was chosen arbitrarily. If you reduce it by too much, it runs slower - O(n^2) to the number of points used.